src/MenuOption.php

```diff
@@ -16,9 +16,9 @@
 use PhpSchool\CliMenu\MenuItem\SelectableItem;
 
 /**
- * This is an Laravel Console Menu Option implementation.
+ * This is a Laravel Console Menu Option implementation.
  */
-class MenuOption extends SelectableItem
+class MenuOption extends SelectableItem implements SelectedOption
```

Comment:
Since now we may have more than one option we should rename `MenuOption`  to `Option` only.

Reply:
As spoken on slack, kept it simple, removing the interface.